lang/main.json

```diff
@@ -428,9 +428,14 @@
     },
     "invite": {
         "addPassword": "Add password",
+        "dialInNumbers": "Dial in telephone numbers",
+        "errorFetchingNumbers": "Could not get numbers",
         "hidePassword": "Hide password",
         "inviteTo": "Invite people to __conferenceName__",
+        "loadingNumbers": "Loading...",
         "locked": "This call is locked. New callers must have the link and enter the password to join.",
+        "noNumbers": "No numbers available",
+        "numbersDisabled": "Dialing in has been disabled",
```

Comment:
Could you please add screenshots with some of the above error messages, so that we can see where and how they appear?